README.md

```diff
@@ -1,19 +1,47 @@
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
 # hypatia
 
 <!-- badges: start -->
-[![R build status](https://github.com/mrc-ide/hypatia/workflows/R-CMD-check/badge.svg)](https://github.com/mrc-ide/hypatia/actions)
-[![CodeFactor](https://www.codefactor.io/repository/github/mrc-ide/hypatia/badge)](https://www.codefactor.io/repository/github/mrc-ide/hypatia)
-[![codecov.io](https://codecov.io/github/mrc-ide/hypatia/coverage.svg?branch=main)](https://codecov.io/github/mrc-ide/hypatia?branch=main)
+
 <!-- badges: end -->
 
+The goal of hypatia is to enable SQUIRE to be run on an individual basis
+rather than aggregate
+
 ## Installation
 
-To install `hypatia`:
+You can install the released version of hypatia from
+[CRAN](https://CRAN.R-project.org) with:
+
+``` r
+install.packages("hypatia")
+```
+
+And the development version from [GitHub](https://github.com/) with:
 
-```r
-remotes::install_github("mrc-ide/hypatia", upgrade = FALSE)
+``` r
+# install.packages("devtools")
+devtools::install_github("mrc-ide/hypatia")
 ```
 
-## License
+## UMDER CONSTRUCTION
+
+\#This is a basic example which shows you how to solve a common problem:
+
+``` r
+library(hypatia)
+## basic example code
```

Comment:
This looks like a dangling thing.

Are we ever gonna write this? We should either have an example or remove the section.